onboarder/installation/index.rst

```diff
@@ -102,31 +102,31 @@ Here are two examples in order to define environment variables:
++----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| ONBOARDER_TOPIC_NAME_FOR_CONSUMPTION               | Pub/Sub topic name to receive messages from the middleware.                                                                                |
++----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| ONBOARDER_QUEUE_NAME                               | Pub/Sub queue name.                                                                                                                        |
++----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| ONBOARDER_CLOUD_STORAGE_BUCKET_NAME                | Identifier of the bucket used to share files between your PIM and the middleware.                                                          |
++----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------+
 
 .. note::
 
-    The values for the variables prefixed by ``ONBOARDER_`` are generated. You can find them in the Partners Portal, under the "Properties" tab of your Onboarder project.
+    The values for the variables prefixed by ``ONBOARDER_`` are generated (except ``ONBOARDER_GOOGLE_APPLICATION_CREDENTIALS``). You can find them in the Partners Portal, under the "Properties" tab of your Onboarder project.
 
 **Optional variables**
 
-+----------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| PUBSUB_EMULATOR_HOST | Use this if you want to use a Pub/Sub emulator during development. In this case you don't need the serviceAccount.json file neither set the ONBOARDER_GOOGLE_APPLICATION_CREDENTIALS variable. |
-+----------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
++----------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| PUBSUB_EMULATOR_HOST | Use this if you want to use a Pub/Sub emulator during development. In this case you don't need the onboarderServiceAccount.json file neither set the ONBOARDER_GOOGLE_APPLICATION_CREDENTIALS variable. |
```

Comment:
In this case you don't need the ``onboarderServiceAccount.json``

Reply:
Thanks but I don't understand what you propose, to remove words after `onboarderServiceAccount.json` ?